.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Security vulnerabilities
+    url: https://github.com/freedomofpress/dangerzone?tab=security-ov-file
```

Comment:
Not sure what this link does, to be honest.

Reply:
This links to a potential `security.md` file. I believe we should remove this for now until we have a better way to report security issues, as discussed on last meeting. I've removed this for now in 2827f789dfb54b633e822ca63c76d893deb9e9a6